drivers/windows/overlay/joinleave_windows.go

```diff
@@ -93,6 +93,10 @@ func (d *driver) EventNotify(etype driverapi.EventType, nid, tableName, key stri
 	d.peerAdd(nid, eid, addr.IP, addr.Mask, mac, vtep, true)
 }
 
+func (d *driver) DecodeTableEntry(tablename string, key string, value []byte) (string, map[string]string) {
+	return "", nil
+}
```

Comment:
Windows overlay seem to be using providerAddress for VTEP which is set differently..
```
func (d *driver) convertToOverlayNetwork(v *hcsshim.HNSNetwork) *network {
        n := &network{
                id:              v.Name,
                hnsId:           v.Id,
                driver:          d,
                endpoints:       endpointTable{},
                subnets:         []*subnet{},
                providerAddress: v.ManagementIP,
        }
```

I would prefer leaving this stub for now and @msabansal address it. This can be done as a bug fix.